plugin-cgroupv2/src/k8s/utils.rs

```diff
@@ -66,6 +66,11 @@ fn list_metric_file_in_dir(
                 .to_str()
                 .with_context(|| format!("Filename is not valid UTF-8: {:?}", path))?;
 
+            // If the dir_uid doesn't end with .slice, we will not find the right files
+            if !(dir_uid.ends_with(".slice")) {
```

Comment:
IMO you can remove the extra parens 
```suggestion
            if !dir_uid.ends_with(".slice") {
```